docs/products/kafka/howto/configure-log-cleaner.rst

```diff
@@ -0,0 +1,34 @@
+Configure log cleaner for topic compaction
+==========================================
+
+The log cleaner is the process ensuring only the most recent value for a certain message key is kept within a partition for :doc:`compacted topics <../concepts/log-compaction>`.  In Aiven for Apache Kafka the log cleaner is enabled, but the log compaction is disabled by default. To enable log compaction, follow these steps:
+
+#. Log into the `Aiven web console <https://console.aiven.io/>`_ and select your Aiven for Apache Kafka service.
```

Comment:
I like that those steps is consistent across the files. I mean how the format is. 💯